attributes/default.rb

```diff
@@ -5,15 +5,14 @@
 default['flink']['user_id']                          = '1510'
 default['flink']['user-home']                        = "/home/#{node['flink']['user']}"
 
-default['flink']['version_base']                     = "1.14"
-default['flink']['version_bugfix']                   = "3.1"
+default['flink']['version_base']                     = "1.17"
+default['flink']['version_bugfix']                   = "1.0"
```

Comment:
I can't find the branch for this build in the `logicalclocks/flink` repository.

Reply:
done https://github.com/logicalclocks/flink/pull/7